prompttools/playground/playground.py

```diff
@@ -54,6 +54,11 @@
             model = st.selectbox("Model", ("text-davinci-003", "text-davinci-002", "code-davinci-002"))
             api_key = st.text_input("OpenAI API Key")
         temperature = st.slider("Temperature", min_value=0.0, max_value=1.0, value=0.5, step=0.01, key="temperature")
+        top_p = st.slider("Top_P", min_value=0.0, max_value=1.0, value=0.5, step=0.01, key="top_p")
+        max_tokens = st.slider("Max Tokens", min_value=0, max_value=100, value=100, step=5, key="max_tokens")
```

Comment:
Could you use a `number_input` for `max_tokens` instead of a slider? Also, there should be no max value for `max_tokens`.

Reply:
Done! I've set the default slider value to 100, min to 0 and no max value.